assets/src/scss/components/_buttons.scss

```diff
@@ -40,7 +39,6 @@
   &:hover,
   &:active,
   &:visited {
-    color: $white;
```

Comment:
I removed this because otherwise it applied to the new button style class, where I just want the same color on hover. It should not be a problem as all buttons use a style.